test/parallel_api/ranges/std_ranges_stable_sort.pass.cpp

```diff
@@ -24,8 +24,10 @@ main()
 
     auto sort_stable_checker = TEST_PREPARE_CALLABLE(std::ranges::stable_sort);
 
-    test_range_algo<0>{big_sz}(dpl_ranges::stable_sort, sort_stable_checker);
-    test_range_algo<1>{}(dpl_ranges::stable_sort, sort_stable_checker, std::ranges::less{});
+    test_range_algo<0, int, data_in, DevicePolicy>{big_sz}(dpl_ranges::stable_sort, sort_stable_checker);
```

Comment:
Effectively, these are 7 different signatures / options for the ranges API.  
Should we be fully eliminating the host policies test for this simplest 2 signatures?

It seems what we actually want might be to just limit / scale the max elements tested based on policy type.

Reply:
Thanks, I've made a change to keep the coverage.

Reply:
An approach with scaling the sizes for the policies also looks good.

It must be simpler. For example we can pass multiple sizes instead of single `max_n`: one for serial policies, one for par policies and one for device policies. Or we can pass coefficients for it to be "scaling" as suggested.

Reply:
I'm OK with either strategy.  What you have now works for me, but I agree that it may result in simpler code to merely pass different max_n for different policies.

Reply:
Could you check #2081? @MikeDvorskiy, you may be interested too.
Let's then decide what which way to follow.